api/tests/integration/infrastructure/repositories/finalized-session-repository_test.js

```diff
@@ -96,4 +96,70 @@ describe('Integration | Repository | Finalized-session', () => {
       });
     });
   });
+
+  describe('#findPublishableSessions', () => {
+
+    afterEach(() => {
+      return knex('finalized-sessions').delete();
+    });
+    context('When there are publishable sessions', () => {
+      it('Finds a list of publishable finalized session', async () => {
```

Comment:
**remarque useless de la nazie de la syntaxe 😅 😈 ** 
- En général on met une ligne vide avant un contexte
- Côté API on ne met généralement pas de majuscules aux descriptifs des `context` et des `it`